translations/readme/README.cat.md

```diff
@@ -1,5 +1,12 @@
 # 🟡 GROC &middot; ![Mida del repositori a GitHub][1] ![Mida del codi][2]
 
+## Llegeix-ho en [altres idiomes](translations/readme/).
+
+![🇬🇧 English][3] ·
```

Comment:
Translate the languages list into its own language, but keep this order (as if they were showed in English).